Source/Metrics/MetricsEngine.swift

```diff
@@ -94,23 +94,23 @@ class MetricsEngine {
         }
     }
     
-    private func postMetrics(payload: RequestParameter, completionHandler: CompletionHandlerType?) {
+    private func postMetrics(_ payload: RequestParameter, completionHandler: CompletionHandlerType?) {
         MetricsClient().post(payload) {
-            (response: ServiceResponse<AnyObject>) in
+            (response: ServiceResponse<Any>) in
             switch response.result {
-            case .Success:
+            case .success:
                 Logger.info("Success: post metrics")
                 completionHandler?(true)
                 
-            case .Failure(let error):
-                Logger.error("Failure: \(error.localizedFailureReason)")
+            case .failure(let error):
+                Logger.error("Failure", error: error)
```

Comment:
Will it impact the log message after using error instead of error.localizedFailureReason?